src/app/AppContext.jsx

```diff
@@ -27,12 +28,59 @@ export function AppProvider({children}) {
         }
     }, [setAdmin, adminRole])
 
+    const [initial, setInitial] = useState()
+    const [version, setVersion] = useState()
+    const [initalMinVersion, setInitialMinVersion] = useState()
+    const [updateRequired, setUpdateRequired] = useState(false)
+    const [error, setError] = useState(false)
+    const updateAvailable = initial && version && initial !== version
+
+    const checkVersion = async first => {
+        try {
+            const response = await fetch('/version.json', {cache: 'no-cache'})
+            const {version: newVersion} = (await response.json())
+
+            const min = await fetch('/minVersion.json', {cache: 'no-cache'})
```

Comment:
Can we just add this property to the existing file, so we don't have to double requests on version checks?

Reply:
Makes sense, let me work on it.